src/databricks/labs/ucx/hive_metastore/tables.py

```diff
@@ -544,13 +544,29 @@ def _all_databases(self) -> list[str]:
                 return list(self._iterator(self._external_catalog.listDatabases()))
             return self._include_database
         except Exception as err:  # pylint: disable=broad-exception-caught
+            if "py4j.security.Py4JSecurityException" in str(err):
+                logger.error(
+                    "Failed to list databases due to Py4JSecurityException. "
+                    "Ensure that the cluster is configured correctly."
```

Comment:
`Ensure that the cluster is configured correctly.` - how? you should not assume that all users know the answer. option here is to update UCX. or reinstall it as the last resort.